helm/prometheus-operator-app/templates/prometheus/service.yaml

```diff
@@ -6,6 +6,7 @@ metadata:
   namespace: {{ template "kube-prometheus-stack.namespace" . }}
   labels:
     app: {{ template "kube-prometheus-stack.name" . }}-prometheus
+    giantswarm.io/monitoring: "true"
```

Comment:
What about also adding the annotation `giantswarm.io/monitoring-app-label: {{ .Chart.Name }}`

See https://intranet.giantswarm.io/docs/monitoring/monitoring_of_apps/#service-object-specification

Reply:
Currently the `app` shows was `app="prometheus-operator-app-prometheus"`, do you feel like this is not enough ?

Reply:
The annotation makes the `app` label in prometheus stable. If an user changes the name the `app` label will change when its left like this.